_docs_use-cases/use-case-consumption-execute-an-iqlquery.md

```diff
@@ -30,7 +30,7 @@ properties:
   - created_at:
   - changed_at:
   - api_route_regex: POST /api/v2/Attributes/ExecuteIQLQuery
-  - published:
+  - published: default
```

Comment:
Where is the difference when it is added default. The excel makes this automatically I think, but changes something?

Reply:
No, this doesn't change anything. :)